Cargo.toml

```diff
@@ -155,7 +155,11 @@ features = [
   "string_pad",
   "replace",
   "cov",
-  "http"
+  "http",
```

Comment:
Oooh.. you took these out here and now you're adding them back, so inadvertently took away features here.

https://github.com/pola-rs/nodejs-polars/pull/253/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L158-L162